### PR TITLE
Spitfire 2: Geneva Conventions Be Damned

### PR DESCRIPTION
**Pull request by swifterjack**
## About The Pull Request

Completely remaps the Spitfire to be a lot more polished, finished, and better. Respecs it into a Botany-Mining-Medical ship, with faction alignment of Syndicate NSV-Gorlex, and a singular GEC Engineer. Also renames the ship to the Geneva class, because it makes no sense to be called the spitfire. 
geneva

## Why It's Good For The Game

Spitfire sucked before, now its pretty!

## Changelog
:cl:
add: Job datums for NSV Medical
change: Remapped Spitfire, renamed to Geneva
balance: Spitfire is now a Botany-Medical-Mining ship.
/:cl:

### CODE REVIEW

**Comment by Martinpachu** [comment]
spitfirent

**Comment by swifterjack** [comment]
could we get this testmerged

**Comment by UltraNumeron** [comment]
Said this in discord but realized its probably better to say my shitty feedback in the pr, theres no roller beds which kinda sucks for a rescue ship, and no holopad so you cant call the medical ship for medical help. Also a grinder for chem would be pretty good.
And less necesarry but still nice would be some science goggles so you can check reagents on the go or just a chem closet.

**Comment by swifterjack** [comment]
it has two holopads, in the bridge andin the surgery room, chemist or engi spawns with hte board for grinder in their locker

**Comment by UltraNumeron** [comment]
Ah alright thanks.

**Comment by UltraNumeron** [comment]
One more thing, currently seemingly no way to close shutters or blast doors or whatever on the back of the ship where the surgery room is so mobs can aggro on you while youre busy doin surgery and bust in your ship.

**Comment by swifterjack** [comment]
> One more thing, currently seemingly no way to close shutters or blast doors or whatever on the back of the ship where the surgery room is so mobs can aggro on you while youre busy doin surgery and bust in your ship.

ah, i forgot window shutters there, fixing. thanks

File: code/modules/jobs/job_types/botanist.dm [hunk 1]
```diff
@@ -34,4 +34,16 @@
 	satchel = /obj/item/storage/backpack/satchel/hyd
 	courierbag = /obj/item/storage/backpack/messenger/hyd
 
+//shiptest!!!!!!!!!!
+/datum/outfit/job/botanist/syndicate/nsv
+	name = "Botanist-Chemist (NSV-M)"
+
+	uniform = /obj/item/clothing/under/syndicate/skirt
```
Comment (MarkSuckerberg) [inline code/modules/jobs/job_types/botanist.dm]: jumpsuit prefs exist for a reason, I know it's funny but you probably expected this

File: _maps/shuttles/shiptest/syndicate_geneva.dmm [hunk 2]
```diff
@@ -0,0 +1,2531 @@
...
+/area/ship/crew/hydroponics)
+"eh" = (
+/obj/effect/spawner/structure/window/reinforced,
+/obj/machinery/door/poddoor/preopen{
+	id = "spitfireshutters"
+	},
+/turf/open/floor/plating,
+/area/ship/crew/dorm)
+"ej" = (
+/obj/structure/sink/puddle,
```
Comment (Firefox13) [inline _maps/shuttles/shiptest/syndicate_geneva.dmm:257]: Out of all things (sink, high-capacity water tank, what-have-you) you put a puddle on a spaceship?

Comment (TheNeoGamer42) [reply _maps/shuttles/shiptest/syndicate_geneva.dmm:257]: > Out of all things (sink, high-capacity water tank, what-have-you) you put a puddle on a spaceship?

Yes.

Comment (swifterjack) [reply _maps/shuttles/shiptest/syndicate_geneva.dmm:257]: this is not changing

Comment (Firefox13) [reply _maps/shuttles/shiptest/syndicate_geneva.dmm:257]: Why

Comment (Ms-Mee) [reply _maps/shuttles/shiptest/syndicate_geneva.dmm:257]: After all, why not? Why shouldn't it be a puddle?

File: code/modules/jobs/job_types/botanist.dm (hunk 1 above)
Comment (swifterjack) [reply code/modules/jobs/job_types/botanist.dm]: dont see the issue

File: _maps/shuttles/shiptest/syndicate_geneva.dmm (hunk 2 above)
Comment (swifterjack) [reply _maps/shuttles/shiptest/syndicate_geneva.dmm:257]: the solgovvie wants their little splishy pool

Comment (Firefox13) [reply _maps/shuttles/shiptest/syndicate_geneva.dmm:257]: Doesn't really fits the theme of a spaceship but I guess your reasoning can't be beaten

> solgovvie

but it's a syndicate ship

Comment (swifterjack) [reply _maps/shuttles/shiptest/syndicate_geneva.dmm:257]: nsv is from solgov and is natural science, Lore Not Read

File: code/modules/jobs/job_types/botanist.dm (hunk 1 above)
Comment (MarkSuckerberg) [reply code/modules/jobs/job_types/botanist.dm]: not being merged until you fix it and I know you know what I mean

Comment (MarkSuckerberg) [reply code/modules/jobs/job_types/botanist.dm]: and that's a promise

Comment (swifterjack) [reply code/modules/jobs/job_types/botanist.dm]: fair enough

**Comment by miniusAreas** [comment]
from Shiptest Shiplist Review, posted in #shipyard 

Spitfire-Class (now Genevia, but with gripes for it)

For a syndicate ship, missing the plasteelium hull that signifies those. Otherwise, ditch botany for the old Spitfire Chemfactory setup, please. For a syndicate medical ship, it doesn’t fit to have botany!

just a small bit of feedback to pick your brain.

**Comment by terg-but-ius** [comment]
> For a syndicate ship, missing the plasteelium hull that signifies those.

I always assumed the ship just having regular, not even reinforced walls was meant to be a weakness of the ship.
PHENOMENAL CHEMICAL AND MEDICAL POWER
flimsy as paper hull

**Comment by miniusAreas** [comment]
> I always assumed the ship just having regular, not even reinforced walls was meant to be a weakness of the ship. PHENOMENAL CHEMICAL AND MEDICAL POWER flimsy as paper hull

correct me if I'm wrong but A - pretty sure plastelium walls are just as flimsy as iron ones, maybe better bomb resistance?

and B - _It still is deconstructed in a welder-wrench-screwdriver operation either way._

File: code/modules/jobs/job_types/botanist.dm (hunk 1 above)
Comment (swifterjack) [reply code/modules/jobs/job_types/botanist.dm]: @MarkSuckerberg fixed it, they are just regular toitlenecks now

**Comment by swifterjack** [comment]
the only issue keeping it from merged is fixed, lgtm girls

File: _maps/shuttles/shiptest/bar_ship.dmm [hunk 3]
```diff
@@ -2282,9 +2282,7 @@
 /obj/machinery/door/firedoor/border_only{
 	dir = 8
 	},
-/obj/machinery/door/airlock/freezer{
-	name = "freezer airlock"
-	},
+/obj/machinery/door/airlock/freezer,
```
Comment (MarkSuckerberg) [inline _maps/shuttles/shiptest/bar_ship.dmm:2285]: Any reason it touches this file?

Comment (swifterjack) [reply _maps/shuttles/shiptest/bar_ship.dmm:2285]: no, actually. i didn't touch it at all, it might have been an odd pull? not really sure

**Pull request merged**